maths-club-pack/content/student/colouring-maps.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Colouring Maps
+type: problem
+printOrder: n/a
+added: 2021-01
+---
+
+# Colouring Maps
+
+Start by drawing an empty square:
+
+<img src="../../images/colouring-maps-1.png" width=200>
+
+Create a *map* by dividing the square into seperate regions. For example:
+
+<img src="../../images/colouring-maps-2.png" width=200>
+
+Now colour in your map such that there are no regions of the same colour which share an edge.
+
+Try to find the smallest number of different colours which you need to be able to do this.
+
+What about other maps, can you find a certain number of colours which are able to colour in any map?
```

Comment:
Could you maybe add a small example of a correctly coloured and incorrectly coloured group of sections (doesn't have to be the full image, could just be a small part of it)?

Also perhaps you could add a comment to say - "If you don't have colours, you can try shading in different ways such as with dots, crosses or stripes" 
(this could be useful in schools were students only have access to paper and pencil, or perhaps a teacher using a chalkboard with only white chalk)